src/visualizers/Turtle.ts

```diff
@@ -84,15 +74,33 @@ positionally to the domain elements. Range and domain must be the same length.
 - strokeColor: The color used for drawing the path.
      **/
     strokeColor: {
-        default: '#ff0000',
+        default: '#c98787',
         type: ParamType.COLOR,
         displayName: 'Stroke Color',
         required: true,
     },
+    /** md
+- stepSize: a number. Gives the length of the segment drawn for each entry.
+     **/
+    stepSize: {
+        default: 20,
+        type: ParamType.INTEGER,
+        displayName: 'Step Size',
+        required: false,
+    },
+    /**
+- start: x,y coordinates of the point where drawing will start
+     **/
+    start: {
+        default: new p5.Vector(),
+        type: ParamType.VECTOR,
+        displayName: 'Start',
+        required: false,
+        description: 'coordinates of the point where drawing will start',
+        hideDescription: true,
+    },
 } as const
 
-// Turtle needs work
-// Throwing the same error on previous Numberscope website
 class Turtle extends P5Visualizer(paramDesc) {
     static category = 'Turtle'
     static description =
```

Comment:
As long as we are doing this, please remove the unnecessary constructor that just calls super, and then you can also delete the import of GenericParamDescription.

Reply:
done